jboss_wildfly/readme.md

```diff
@@ -0,0 +1,46 @@
+# jBoss-Wildfly template for InfluxDB v2
+
+Provided by: Ignacio Van Droogenbroeck
+
+This Dashboard offers you information about your jBoss-Wildfly instance. Uptime, CPU Usage, Heap Memory, Non Heap Memory, Memory Pool Collections, Networking and Disk IO.
+
+![Dashboard Screenshot](screenshot.png)
+
+### Quick Install
+
+If you have your InfluxDB credentials [configured in the CLI](Vhttps://v2.docs.influxdata.com/v2.0/reference/cli/influx/config/), you can install this template with:
+
+```
+influx apply -u https://raw.githubusercontent.com/influxdata/community-templates/master/jboss_wildfly/jboss_wildfly.yml
```

Comment:
Your file name is `jboss-wildfly.yml` with a hyphen instead of an underscore as it uses here

Reply:
It's done. Thank you for this correction :+1: